.github/workflows/tests-php.yml

```diff
@@ -3,24 +3,34 @@ on:
   push:
 jobs:
   test:
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        suite:
-          - wpunit --env singlesite
-          - wpunit --env multisite
-          - muwpunit
-    runs-on: ubuntu-latest
+        # Update this as WordPress releases new backward patches: https://wordpress.org/download/releases/
+        # Currently supporting two releases back.
+        wordpress:
+          - latest
+          - '6.3.3'
+          - '6.2.4'
+
+        php:
+          - '7.4'
+          - '8.0'
+          - '8.1'
+          - '8.2'
+
+    name: "Tests: WP ${{ matrix.wordpress }} / PHP ${{ matrix.php }}"
```

Comment:
Is this matrix going to spin up 12 different instances?

Reply:
Yep, that's correct.